android/app/src/main/res/values-es/strings.xml

```diff
@@ -1446,8 +1453,11 @@
 	<string name="type.leisure.track.area">Pista deportiva</string>
 	<string name="type.leisure.water_park">Parque acuático</string>
 	<string name="type.leisure.beach_resort">Resort de playa</string>
+	<string name="type.man_made">Construcción artificial</string>
```

Comment:
It should be only "Construcción" (all constructions are artificial)

Reply:
@muralito Heh, you've got a point :) but this string is meant to highlight the opposite idea of "natural". In JOSM, this is just "Artificial"; the direct translation of "man-made" is "antropogénico", which is Greek and not ideal :)

Reply:
Changed to just "Construcción".

Please check [PR 7558](https://github.com/organicmaps/organicmaps/pull/7558).